NuclearData/ceNeutronData/aceDatabase/aceNeutronDatabase_class.f90

```diff
@@ -51,19 +52,26 @@ module aceNeutronDatabase_class
   !! It's possible to use probability tables in the unresolved resonance range if
   !! ures is included in the input file
   !!
+  !! NOTE: the unionised majorant is not calculated and used if probability tables 
```

Comment:
I think we should open an issue on this. It will be very desirable to still have a unionised majorant, e.g., if we want to accurately run fast problems. I understand it shouldn't have a big impact for thermal problems, even with burnup.

Reply:
Sure, I'll open an issue. I am not sure if one can a priori find out what's the majorant when using probability tables. One approach could be to sample, say, 1000 times per each energy when calculating the majorant, and retaining the highest xs.